packages/manager/apps/dedicated/client/app/ip/byoip/byoip.html

```diff
@@ -36,6 +36,8 @@
         </oui-step-form>
         <!-- Region -->
         <oui-step-form
+            data-on-focus="$ctrl.loadCampuses($ctrl.byoip.ipRir)"
```

Comment:
Optional: can you initiate this call as soon as ipRir is selected? this will improve improve load time and user experience